sphinx/ext/todo.py

```diff
@@ -209,6 +210,17 @@ def depart_todo_node(self, node):
     self.depart_admonition(node)
 
 
+def latex_visit_todo_node(self, node):
+    # type: (nodes.NodeVisitor, todo_node) -> None
+    title = node.pop(0).astext().translate(tex_escape_map)
+    self.body.append('\n\\begin{sphinxadmonition}{note}{%s:}' % title)
```

Comment:
I guess as potential Unicode chars all come from variable `title` they may be already ok even without `u` ? (this just indicates how ignorant I am)